static/app/views/performance/transactionSummary/transactionReplays/transactionReplays.tsx

```diff
@@ -143,7 +143,7 @@ function ReplaysContent({
   const {replays, isFetching, fetchError} = useReplayList({
     eventView,
     location,
-    organization,
+    path: `/organizations/${organization.slug}/replays/`,
```

Comment:
@cmanallen lmk the new url and we'll update this line

Reply:
Set the `queryReferrer` parameter to `issueReplays`.

Reply:
but the file is `transactionReplays`...